development/accessibility.md

```diff
@@ -0,0 +1,44 @@
+# Accessibility
+Accessibility is makimg services that everyone can use regardless of their ability or circumstance.
+
+## Why should we care?
+All public sector digital services are required by law to meet the AA standard of the [W3C Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/).
+
+It also means that we are not restricting our services and they can be used by everyone.
+
+## How are we doing this
+We are doing this by building with accessibility in mind from the very start and doing usability testing throughout the project.
+
+**When developing a service keep in mind the following:**
+1. 	All functionality must be usable with the keyboard
+
+ **Example:** pop-up calendars should also let users type in a date
+
+2. HTML is valid (using a [validator service](https://validator.w3.org/))
+    * Have complete start and end tags, are nested, according to their specs, do not contain duplicate attributes, any IDs are unique
+
+3. If the web content sets a timer, the user must have the option to adjust/extend it
+
+4. A mechanism to allow the users to exit/remove repetitive blocks of content 
+
+ **Example:** Skip to content at the top of the page to bypass common nav bar
+
+5. If an input error is automatically detected, it can be identified and described to the user in text
+
+6. Text can be resized (without assistive technology) up to 200% without loss of content or functionality
```

Comment:
I initially read this as, the website should include buttons to resize text, it'd be good to clarify it

Reply:
We have taken this to mean using ctrl / cmd + scroll will keep the content visible without scrolling to the right

I'll explain this too.